Adabot_Operation_Game/adabot_operation_game.py

```diff
@@ -0,0 +1,42 @@
+# Adabot Operation Game
+#  CPX, alligator clips, copper tape, tweezers, surgery, and fun!
+
+import board
+import touchio
+from adafruit_circuitplayground.express import cpx
+# import time  # uncomment if testing raw read values
+
+cap_pins = (board.A1, board.A2, board.A3, board.A4, board.A5,
+            board.A6, board.A7)
+touch_pads = []
+for i in range(7):
+    touch_pads.append(touchio.TouchIn(cap_pins[i]))
+for touch_pad in touch_pads:
+    touch_pad.threshold = 3500  # adjust value to fine-tune touch threshold
+
+MAGENTA = (10, 0, 10)
+VIOLET = (5, 0, 15)
+BLUE = (0, 0, 20)
+CYAN = (0, 10, 10)
+GREEN = (0, 20, 0)
+YELLOW = (10, 10, 0)
+ORANGE = (15, 5, 0)
+RED = (20, 0, 0)
+WHITE = (3, 3, 3)
+
+COLORS = [MAGENTA, VIOLET, BLUE, CYAN, GREEN, YELLOW, ORANGE, RED, WHITE]
+
+cpx.pixels.fill((WHITE))
```

Comment:
Since the variables you created have one set of parens already, I'm not sure that two sets are needed here in this line. Try `cpx.pixels.fill(WHITE)` to see if it works.

Reply:
great catch, thanks. I dropped the extra parens and it works. Will re-commit w that change.